### ISSUE TITLE
Use LLM training data to find best layout

### ISSUE THREAD

**Issue by kyranmcdonnell** [AtomicFrontierCode/keyboards#14]
I realised an ideal dataset for this would using ones for training LLMs. E.g. https://kili-technology.com/large-language-models-llms/9-open-sourced-datasets-for-training-large-language-models

A cool project would be to find the optimal layout for different spoken languages, in addition to coding languages which are also included in the LLM datasets.

It would be interesting to create an open source 'base layout' for writing, coding, and writing+coding, which then allow for 'fine tuning' using a person's code and/or writing.

**Comment by cengbrecht** [comment]
I agree with your idea. I think that would be very cool, I was actually thinking that my typing for normal text vs my coding text would be significantly different. Being able to create a coding layout, vs a writing layout would be interestingly helpful if they were significantly different from each other.
